install.sh

```diff
@@ -214,7 +214,7 @@ install_packages( )
 #     yum -y update brings *all* installed packages up to date
 #     without seeking user approval
 #     elevate_if_not_root yum -y update
-      if [[ "${VERSION_ID}" == "8" ]]; then
+      if (( "${VERSION_ID%%.*}" >= "8" )); then
```

Comment:
A couple notes on how this works. `${VERSION_ID%%.*}` is a [variable expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) for `VERSION_ID` that trims all contents matching the wildcard `.*` from the end of the variable. Note: that's a [glob pattern](https://www.man7.org/linux/man-pages/man7/glob.7.html), not a regex. The `(( ))` allows for [integer arithmetic comparisons](https://www.gnu.org/software/bash/manual/html_node/Shell-Arithmetic.html), rather than string comparisons.